src/pxWindow.h

```diff
@@ -87,7 +87,7 @@ class pxWindow: public pxWindowNative
   // pxSurfaceNative abstracts a platform specific drawing surface
   // to perform platform specific drawing please see pxWindowNative.h
   // for the definition of this type
-  virtual void onDraw(pxSurfaceNative /*s*/) {}
+  virtual void onDraw() {}
```

Comment:
It generates the following error:
```
In file included from /home/sw/projects/pxscene/pxCore/src/glut/../glut/pxConfigNative.h:26:0,
                 from /home/sw/projects/pxscene/pxCore/src/glut/../pxCore.h:72,
                 from /home/sw/projects/pxscene/pxCore/src/glut/pxWindowNative.cpp:5:
/home/sw/projects/pxscene/pxCore/src/glut/../glut/pxWindowNative.h:102:16: error: ‘virtual void pxWindowNative::onDraw(pxSurfaceNative)’ was hidden [-Werror=overloaded-virtual]
   virtual void onDraw(pxSurfaceNative surface) = 0;
                ^~~~~~
In file included from /home/sw/projects/pxscene/pxCore/src/glut/pxWindowNative.cpp:6:0:
/home/sw/projects/pxscene/pxCore/src/glut/../pxWindow.h:90:16: error:   by ‘virtual void pxWindow::onDraw()’ [-Werror=overloaded-virtual]
   virtual void onDraw() {}
                ^~~~~~
cc1plus: all warnings being treated as errors
```
seems all backends needs to be adjusted accordingly.